protected/models/moderator/moderator.object.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * ModelObject Class For Example Model
+ */
+class ExampleObject extends ModelObjectCore
+{
+    /**
+     * @var string Default Data Base Table
+     */
+    public $defaultTableName = 'example';
+
+    /**
+     * @var string Data Base Table For Example Data
+     */
+    public $tableExample = 'example';
```

Comment:
Codacy found an issue: [Member variable "$tableExample" is not in valid snake_case format, try "$table_example"](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)